examples/pendulum_friction-less/model_train_LatentODE.jl

```diff
@@ -34,24 +34,25 @@ include("create_data.jl")
     # diffeq = SPendulum()
     # diffeq = Pendulum_NN_friction()
     # diffeq = NODE(2, augment_dim = 2)
-    diffeq = NODE(16)
+    # diffeq = NODE(16)
+    diffeq = NODE(2)
```

Comment:
For now, let's keep `diffeq = NODE(16)` as in the python implementation.

Reply:
Will make change